src/Artsy/Router/buildServerApp.tsx

```diff
@@ -87,40 +81,36 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
           )
         }
 
-        const { relayData, loadableState } = await trace(
+        const { loadableState, relayData: _relayData, styleTags } = await trace(
           "buildServerApp.fetch",
           (async () => {
-            // Kick off relay requests to prime cache
-            // TODO: Remove the need to do this by using persisted queries.
+            // Kick off relay requests to prime cache. TODO: Remove the need to
+            // do this by using persisted queries.
             ReactDOMServer.renderToString(<ServerApp />)
-            // Serializable data to be rehydrated on client
-            const data = await relayEnvironment.relaySSRMiddleware.getCache()
+            // Extract render queue for bundle split components using dyanamic `import()`
             const state = await getLoadableState(<ServerApp />)
-            return { relayData: data, loadableState: state }
+            // Extract CSS styleTags to inject for SSR pass
+            const tags = collectSSRStyles(<ServerApp />)
```

Comment:
This is basically the only change, aside from docs. It pulls out all of the CSS and returns a string of `<style>` tags that we can embed in the head of our page for server side rendering.